### ISSUE TITLE
Incorrect comment?

### ISSUE THREAD

**Issue by prescod** [hpyproject/hpy#145]
https://github.com/hpyproject/hpy/blob/00b41073288b1abc401954acf86dcb74551654c8/hpy/devel/__init__.py#L41

```
    def get_ctx_sources(self):
        """ Extra sources needed only in Universal mode.
        """
        return list(map(str, self.src_dir.glob('ctx_*.c')))
```

This function seems to be called when I run `proof-of-concept/test_pof.sh wheel cpython` rather than `proof-of-concept/test_pof.sh wheel universal` so the sources seems to be needed only in cpython mode.

**Comment by hodgestar** [comment]
@prescod Thanks! You are correct. I am fixing this in #146.

**Comment by hodgestar** [comment]
Fixed in #146. Thanks again!
